lib/deimos/kafka_source.rb

```diff
@@ -88,8 +88,9 @@ def import_without_validations_or_callbacks(column_names,
                                                   array_of_attributes,
                                                   options={})
         results = super
-        return unless self.kafka_config[:import]
-        return if array_of_attributes.empty?
+        if !self.kafka_config[:import] || array_of_attributes.empty?
+          return ActiveRecord::Import::Result.new([], 0, [], [])
+        end
```

Comment:
Is there a reason you can't just return `results` here?

Reply:
oh good point, will try that out - i forgot that we called `super` haha

Reply:
Yeah - the `results` from `super` works with this too